test/kinetics/kineticsFromScratch3.cpp

```diff
@@ -136,11 +136,11 @@ TEST_F(KineticsFromScratch3, add_plog_reaction)
     //                [(100.0, 'atm'), 5.963200e+56, -11.529, 52599.6])
     Composition reac = parseCompString("H2:1, O2:1");
     Composition prod = parseCompString("OH:2");
-    std::multimap<double, Arrhenius> rates {
-        { 0.01*101325, Arrhenius(1.212400e+16, -0.5779, 10872.7 / GasConst_cal_mol_K) },
-        { 1.0*101325, Arrhenius(4.910800e+31, -4.8507, 24772.8 / GasConst_cal_mol_K) },
-        { 10.0*101325, Arrhenius(1.286600e+47, -9.0246, 39796.5 / GasConst_cal_mol_K) },
-        { 100.0*101325, Arrhenius(5.963200e+56, -11.529, 52599.6 / GasConst_cal_mol_K) }
+    std::multimap<double, ArrheniusBase> rates {
+        { 0.01*101325, ArrheniusBase(1.212400e+16, -0.5779, 10872.7 * 4184.0) },
```

Comment:
You're anticipating that `ArrheniusBase` will become just `Arrhenius` eventually, right? I really hope this change from temperature units to energy units won't catch too many people by surprise.

Reply:
This is correct (I may rename `ArrheniusBase` to `Arrhenius3` to make this clear). I'm not sure that many people instantiate `Arrhenius` directly, and it would be odd to have a default unit that is different from the YAML specification.

Reply:
You're right that this is probably not the most common means of instantiating `Arrhenius` objects for end-users. One thing that would be simple to do, though not foolproof, would be to call out this difference in the Doxygen deprecation note that describes `ArrheniusBase` as the replacement. Though right now `Arrhenius2` is still missing such a marking.

Reply:
I think that is appropriate. Once `InterfaceKinetics` is done, there won't be a problem raising noisy warnings, etc..

Reply:
I believe that adding a `todo` in the docstring of the constructor is likely the best option for the moment (which I just added).